app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PasswordsController < Devise::PasswordsController
+  # POST /resource/password
+  def create
+    self.resource = resource_class.send_reset_password_instructions(resource_params)
+    yield resource if block_given?
+
+    if successfully_sent?(resource)
+      render json: {message: 'Reset password email was successfully sent'}, status: :ok
```

Comment:
I think this could be just a string response, @belgamo how are we handling the response bodies on the front side?
Can It be something like: 
```ruby
render json: 'Reset password email was successfully sent', status: :ok
```
?

Reply:
When responding to errors, other controllers have :message on the json
(from requests/companies_spec.rb)
![image](https://user-images.githubusercontent.com/36737050/161318609-ade1ef2a-a976-4bca-bc3c-0e07719eb439.png)

The resource is being sent like that on successful requests. 
(from controllers/companies_controller.rb)
![image](https://user-images.githubusercontent.com/36737050/161318995-955353c0-27a0-4669-8d27-8139839fa3f0.png)

Since on reset password request there isn't a resource, should i send a empty json with status: :ok?.

Reply:
nice catch, let's keep it